test/evergreen/tcmalloc_install_or_build.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#
+# Make a tcmalloc shared library avilable for preloading into the environment.
+
+set -u
+
+if [[ $# -ne 1 ]]; then
+    echo "Usage: $0 build-variant"
+    exit 1
+fi
+
+build_variant=$1
+
+# This is tcmalloc upstream revision 093ba93 source patched by the SERVER team.
+# https://github.com/mongodb-forks/tcmalloc/releases/tag/mongo-SERVER-85737
+PATCHED_SRC=mongo-SERVER-85737
+PATCHED_TGZ="${PATCHED_SRC}.tar.gz"
+PATCHED_TGZ_URL="https://github.com/mongodb-forks/tcmalloc/archive/refs/tags/${PATCHED_TGZ}"
+PATCHED_SRC_DIR="tcmalloc-${PATCHED_SRC}"
+
+# Sensitive variables retrieved through Evergreen so they won't be exposed
+# in source control.
+#
+# Export the AWS_* variables so that they are visible for the 'aws' cli.
+S3_URL=${s3_bucket_tcmalloc}
```

Comment:
`S3_URL` does not seem to be used in this script.

Reply:
Oops! Fixed.